qemu/tests/mq_enabled_chk.py

```diff
@@ -0,0 +1,82 @@
+import logging
+import re
+
+from virttest import env_process
+from virttest import error_context
+from virttest import utils_net
+from virttest import utils_test
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    MULTI_QUEUE enabled by default check test
+
+    1) Boot up a VM, and login to the guest
+    2) Check the queues according to the value set in qemu cmd line for
+       smp and queues
+    3) do ping test
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+
+    def get_queues_status(session, ifname, timeout=240):
+        """
+        Get queues status
+        """
+        mq_get_cmd = "ethtool -l %s" % ifname
+        nic_mq_info = session.cmd_output(mq_get_cmd, timeout=timeout,
+                                         safe=True)
+        queues_reg = re.compile(r"Combined:\s+(\d)", re.I)
+        queues_info = queues_reg.findall(" ".join(nic_mq_info.splitlines()))
+        if len(queues_info) != 2:
+            err_msg = "Oops, get guest queues info failed, "
+            err_msg += "make sure your guest support MQ.\n"
+            err_msg += "Check cmd is: '%s', " % mq_get_cmd
+            err_msg += "Command output is: '%s'." % nic_mq_info
+            test.cancel(err_msg)
+        return [int(x) for x in queues_info]
+
+    def chk_mq_enabled(vm, queues):
+        login_timeout = int(params.get("login_timeout", 360))
+        session = vm.wait_for_login(timeout=login_timeout)
+
+        mac = vm.get_mac_address(0)
+        ifname = utils_net.get_linux_ifname(session, mac)
+        queues_status = get_queues_status(session, ifname)
```

Comment:
You actually get two numbers of queues, so name it `queues_status` might be confusing.